testpilot/frontend/static-src/styles/modules/_buttons.scss

```diff
@@ -59,6 +59,12 @@
     }
   }
 
+  &.warning {
+    background: #ee4b36;
+    border: 1px solid #d92215;
+    color: #fff;
```

Comment:
Ah, actually, yeah, i don't really care about this...let's get the thing merged and i can do a clean up. my guess is that i will need to do a little tidy after all of the survey stuff gets changed to so maybe i should open a tracking bug for myself.